docs/getting_started.md

```diff
@@ -1,58 +1,122 @@
 # Getting started with `build_runner`
 
-`build_runner` can be used as a development server.
+To use `build_runner`, you need a 2.x (dev channel or fresher) version of
+the Dart SDK.
 
-1. Add a `dev_dependency` on `build_runner: ^0.7.0`
-2. Add a `dev_dependency` on `build_web_compilers: ^0.2.0`
-3. Run `pub get`
-4. To start the server run `pub run build_runner serve`
+* [Automated installers](https://www.dartlang.org/install#automated-installation-and-updates)
+* [Direct downloads](https://www.dartlang.org/install/archive#dev-channel)
 
-The first build will be the slowest. After that assets are cached on disk and
-incremental builds will be faster. While the `serve` command is running any
-changes you save to files will trigger rebuilds automatically.
+If you have issues using `build_runner`, see the
+[Troubleshooting section](#troubleshooting), below.
 
-## Available commands
 
-In addition to `serve` you can use:
+## Using `build_runner` as a development server
 
-- `build` - run a single build and exit. This is most useful if your build also
-  generates outputs to your source directory. With `--output <dirname>` this
+1. Edit your package's **pubspec.yaml** file,
+   removing all transformers and
+   adding dev dependencies on **build_runner** and **build_web_compilers**:
+
+   ```yaml
+   ...  
+   environment:
+     sdk: '>=2.0.0-dev <2.0.0'
+   ...  
+   dev_dependencies:
+     build_runner: ^0.7.0
+     build_web_compilers: ^0.2.0
+   ```
+
+2. Get package dependencies:
+
+   ```sh
+   pub get
+   ```
+
+3. Start the server:
+  
+   ```sh
+   pub run build_runner serve
+   ```
+
+While the `serve` command runs, every change you save triggers a rebuild.
+
+The first build is the slowest. After that, assets are cached on disk and
+incremental builds are faster.
+
+## Using other build_runner commands
+
+In addition to **serve** you can use:
+
+- **build:** Runs a single build and exits. This is most useful if your build also
+  generates output to your source directory. With `--output <dirname>` this
   also creates a merged output directory with all sources and generated assets.
-- `watch` - like `build` but reruns after file changes. With
+
+- **watch:** Like `build` but reruns after file changes. With
   `--output <dirname>` the merged output directory will be kept up to date with
   changes. This can be used to keep the outputs updated for use with another
   filed-based development server.
-- `test` - Creates an output directory and runs `pub run test` within it.
 
-## Switching to Dart2Js
+- **test:** Creates an output directory and runs `pub run test` within it.
```

Comment:
Should we call out that `build_test` needs to be a dev dependency for this to work? The latest version of the command will check for it and warn you - but that might be a surprise

Reply:
ya we should add that imo

Reply:
Done.